standard/arrays.md

```diff
@@ -232,5 +232,7 @@ When an array creation expression includes both explicit dimension lengths and a
 > int[] z = new int[3] {0, 1, 2, 3}; // Error, length/initializer mismatch
 > ```
 > Here, the initializer for `y` results in a compile-time error because the dimension length expression is not a constant, and the initializer for `z` results in a compile-time error because the length and the number of elements in the initializer do not agree. *end example*
+<!-- markdownlint-disable MD028 -->
 
+<!-- markdownlint-enable MD028 -->
```

Comment:
@jskeet I added a number of these comments to disable a warning.

There are numerous locations in the standard where we have repeated *note* and *warning* blocks. But we want to enable this warning to prevent blank (non-block-quote) lines that split an example or note.

The word converter passed, but I wanted you to see this.

Reply:
Thanks for the shout-out... I'll double-check the Word converter after merging; if we need to add code to explicitly ignore these and avoid creating warnings, that should be easy to do.

Reply:
(If we can work out a different way of separating consecutive notes/warnings, that would be even better, of course. But this is a good start.)

Reply:
> (If we can work out a different way of separating consecutive notes/warnings, that would be even better, of course. But this is a good start.)

I looked at the markdown lint docs, and I didn't find a better alternative. (Doesn't mean it doesn't exist, just that I didn't find it.)